tests/run-tests.sh

```diff
@@ -14,5 +14,6 @@ echo "Linking $CORE_PACKAGE_DIR to $CORE_GIT_DIR"
 rm -rf $CORE_PACKAGE_DIR
 ln -s $CORE_GIT_DIR $CORE_PACKAGE_DIR
 
-elm-make --yes --output test.js Test.elm
-node test.js
+elm-make --yes --output test.js Main.elm
+
+npm test
```

Comment:
I think that this depend caching (elm-make do not overwrite & recompile anything but will try though). That doesn't feels right:-/ Please pay attention to this line & integration with `elm-test`